### PR TITLE
[DCOS-43919] Release Confluent Kafka 2.4.0-4.1.1

### PR DESCRIPTION
**Pull request by elezar**
## Description
https://jira.mesosphere.com/browse/DCOS-43919

## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/services/confluent-kafka/2.4.0-4.1.1/api-reference/index.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+  }
+]
+```
+
+## Alter Topic Partition Count
+
+```
+$ dcos {{ model.packageName }} --name={{ model.serviceName }} topic partitions topic1 2
+{
+  "message": "Output: WARNING: If partitions are increased for a topic that has a key, the partition logic or ordering of the messages will be affectednAdding partitions succeeded!n"
```
Comment (SarahStegall) [inline pages/services/confluent-kafka/2.4.0-4.1.1/api-reference/index.md]: This message doesn't look right. What does "affectednAdding partitions succeeded!n" mean? Is this correct?

Comment (elezar) [reply pages/services/confluent-kafka/2.4.0-4.1.1/api-reference/index.md]: The `n` stuck in the middle between `addected` and `Adding` should be `\n` indicating a newline. The same holds for the final `n`.

**Comment by elezar** [comment]
@SarahStegall thanks for the review. I have made the changes to the file where you found these, but I'm sure that there could be more of them -- definitely for other package versions -- I will create ticket to track cleaning these up.

File: pages/services/confluent-kafka/2.4.0-4.1.1/api-reference/index.md (hunk 1 above)
Comment (SarahStegall) [reply pages/services/confluent-kafka/2.4.0-4.1.1/api-reference/index.md]: Will fix.

**Pull request merged**